pkg/status/templates/header.tmpl

```diff
@@ -160,4 +160,49 @@ Cluster Checks Dispatching
   Status: unknown
 {{- end }}
 {{- end }}
+
+{{- if .admission }}
```

Comment:
Does it make sense to put this section in a separate template file instead?

Reply:
all the cluster agent sections are in this file (external metrics, cluster checks, leader elections)